sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -273,7 +274,7 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
 
         aggregate.AggUtils.planStreamingAggregation(
           namedGroupingExpressions,
-          aggregateExpressions,
+          aggregateExpressions.map(expr => expr.asInstanceOf[AggregateExpression]),
```

Comment:
Does it means that pandas UDAF doesn't support streaming aggregation?
Maybe we should check that streaming aggregation doesn't contain pandas UDAF and throw exception with reasonable message if it contains.

Reply:
Yeah streaming aggregation won't work. Good point. Let me add the check.

Reply:
Added check.